integrations/pipedrive/integration.definition.ts

```diff
@@ -2,10 +2,10 @@ import { IntegrationDefinition, z } from '@botpress/sdk'
 import { actions } from './definitions'
 
 export default new IntegrationDefinition({
-  name: 'pipedrive',
+  name: 'plus/pipedrive',
```

Comment:
should the workspace name be botpress/ ?  @ermek-botpress how were you envisioning this work from our repo?